i18n/en.pot

```diff
@@ -47,34 +47,28 @@ msgstr ""
 msgid "Search by name / code"
 msgstr ""
 
-msgid "Project funders"
+msgid "Project funders (*)"
```

Comment:
It's more versatile to leave the `(*)` out of the literal so when the required/non-required flag changes we don't need to change the translation.